src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.ClassGroup;
+import seedu.address.model.person.ListCommandPredicate;
+
+/**
+ * Parses input arguments and creates a new ListCommand object
+ */
+public class ListCommandParser implements Parser<ListCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ListCommand
+     * and returns a ListCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ListCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+
+        if (!trimmedArgs.isEmpty()) {
+            String[] classGroups = trimmedArgs.split("\\s+");
```

Comment:
Could I check if it is the intention of the list command to accept multiple classGroup parameters.

If so, could there be a bug if a Person's classGroup contains a space, eg. "F14 3", and user inputs `ls F14 3`, the parser would try to find "F14" and "3" separately.

To solve this, we could either:
1. Enforce that classGroup must not contain whitespace
2. Make the list command: ls c/class group 1 c/class group 2, to distinguish the different classGroups